src/styles/utils/_variables.scss

```
@@ -0,0 +1,4 @@
+$transitionTimeHeader: 300ms;
+$transitionTimeCard: 300ms;
```

Why do you need 2 separate variables for the same value? 
```suggestion
$transition-duration: 300ms;
```

---

in case we wanna have different durations in the future